build.xml

```diff
@@ -390,24 +390,40 @@
         <delete dir="${command-line-html-dir}"/>
         <!-- If you don't want to generate on-line doc for a command, use package-command instead of document-command -->
         <document-command title="AddCommentsToBam"                  main-class="picard.sam.AddCommentsToBam"/>
-        <document-command title="AddOrReplaceReadGroups"            main-class="picard.sam.AddOrReplaceReadGroups"/>
+        <document-command title="AddOrReplaceReadGroups"      main-class="picard.sam.AddOrReplaceReadGroups"/>
```

Comment:
The iml ipr file issue is a persistent problem.  I am working with @droazen to resolve.